### PR TITLE
nss: PPC fixes

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

PPC fixes, which finally resolve a failing build on Rosetta: https://trac.macports.org/ticket/64900

I cannot test `ppc64` now, but from what I understand VSX and Crypto are not supported on G5, since they were introduced in ISA 2.06 and later. See also: https://bugzilla.mozilla.org/show_bug.cgi?id=1687164

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

P. S. Initially I wrote a separate condition for Rosetta:
```
platform darwin 10 {
    if {${build_arch} eq "ppc"} {
    # Otherwise -Di386 flag is added and build fails. See nss/coreconf/Darwin.mk
        build.env-append CPU_ARCH=ppc
    }
}
```
But it seems that the code in Portfile was faulty (with missing definitions for `ppc` and `ppc64`), which led to confusion with archs in build system. Once definitions are added, they are used correctly, and Rosetta-specific code becomes redundant. I have verified the build without it, so removed it.

### CODE REVIEW

**Comment by barracuda156** [comment]
Perhaps the patch can be applied unconditionally – it should not break anything for non-PPC systems. Please let me know if that is preferable.

**Comment by barracuda156** [comment]
@mascguy No maintainer, and I think this is safe to merge. Patch concerns only PPC systems, update benefits all, checks pass.

**Comment by barracuda156** [comment]
@mascguy Apparently someone already done the update in the meanwhile, but PPC fixes are still due.

**Comment by mascguy** [comment]
LGTM. Just update the commit message to close the ticket, and we should be all set.

**Comment by barracuda156** [comment]
> LGTM. Just update the commit message to close the ticket, and we should be all set.

@mascguy Done! Thank you.

**Review by mascguy — APPROVED** [review]
Will merge once CI completes

**Pull request merged**
